src/dom/query.ts

```diff
@@ -86,12 +88,26 @@ function hitTest(node: Element, clientX: number, clientY: number): boolean {
  * ```
  */
 export
-function scrollIfNeeded(area: HTMLElement, elem: HTMLElement, threshold = 0): void {
+function scrollIfNeeded(area: HTMLElement, elem: HTMLElement): void {
```

Comment:
Not sure about this one. Naming it the same as the native method could cause grepping problems, and it's not strictly the same API.